frontend/src/components/animation/index.ts

```diff
@@ -0,0 +1,3 @@
+export { Block } from '@/components/animation/Block';
+export { MotionContainer } from '@/components/animation/MotionContainer';
+export { Reveal } from '@/components/animation/Reveal';
```

Comment:
If you use the relative paths here (or in any index file) it means later you can move the entire directory without having to adjust these.

```suggestion
export { Block } from './Block';
export { MotionContainer } from './MotionContainer';
export { Reveal } from './Reveal';
```